apps/block_scout_web/lib/block_scout_web/views/block_view.ex

```diff
@@ -6,6 +6,7 @@ defmodule BlockScoutWeb.BlockView do
   alias Explorer.Chain
   alias Explorer.Chain.{Address, Block, Wei}
   alias Explorer.Chain.Block.Reward
+  alias Explorer.Counters.{BlockBurnedFeeCounter, BlockPriorityFeeCounter}
```

Comment:
Weird that this is added without being used

Reply:
Yeah, that's because they are used in the [template](https://github.com/celo-org/blockscout/pull/396/files#diff-c1563d86f788ea13f80d8ca304851301c5a51def49fe494c8617beff1350a75fR1).